README.md

```diff
@@ -141,6 +146,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
 - [Webpage Nepal Pvt. Ltd.](http://webpagenepal.com/) | New Road, Pokhara | Pokhara based web development company
+- [WebSoft Nepal Pvt. Ltd.](http://www.websoftnepal.com) | Pashupati Plaza, Indradev Marg, Narayangarh | Provides a complete solution on Web Design and Hostings
```

Comment:
`Pashupati Plaza, Indradev Marg, Narayangarh`

Can you shorten this down to `Intradev Marg, Narayangarh` or `Pashupati Plaza, Narayangarh`?